datadog/dogshell/common.py

```diff
@@ -10,23 +10,33 @@
 
 def print_err(msg):
     if is_p3k():
-        print('ERROR: ' + msg + '\n', file=sys.stderr)
+        print(msg + '\n', file=sys.stderr)
     else:
         sys.stderr.write(msg + '\n')
 
 
 def report_errors(res):
     if 'errors' in res:
-        for e in res['errors']:
-            print_err('ERROR: ' + e)
+        errors = res['errors']
+        if isinstance(errors, str):
```

Comment:
Let's do the following instead, so that we always print, even if the type is not `list` or `str`
```python
if isinstance(errors, list):
    for ...
else:
    print_err(...)
```